sklearn/cluster/_affinity_propagation.py

```diff
@@ -382,8 +382,8 @@ def fit(self, X, y=None):
 
         Parameters
         ----------
-        X : array-like or sparse matrix, shape (n_samples, n_features), or \
-            array-like, shape (n_samples, n_samples)
+        X : {array-like, sparse matrix} of shape (n_samples, n_features), or \
+            array-like of shape (n_samples, n_samples)
```

Comment:
Add an over-indented here for visual indent.

Reply:
ok